tests/playground.html

```diff
@@ -586,6 +591,7 @@
           </shadow>
         </value>
       </block>
+      <label text="Input/Output:" web-style="ioLabel"></label>
```

Comment:
web-style?  className would be more canonical.  Or if you want a web prefix, maybe web-class?  But I don't really know what I'm talking about since I've no idea what's going on in Android and iOS land.  It's just that 'style' would imply content like 'color: red' whereas 'class' implies content like 'myButton'.